ErsatzTV.Application/Channels/Commands/RefreshChannelListHandler.cs

```diff
@@ -81,7 +81,8 @@ await foreach (ChannelResult channel in GetChannels(dbContext).WithCancellation(
                 ChannelName = channel.Name,
                 ChannelCategories = GetCategories(channel.Categories),
                 ChannelHasArtwork = !string.IsNullOrWhiteSpace(channel.ArtworkPath),
-                ChannelArtworkPath = channel.ArtworkPath
+                ChannelArtworkPath = channel.ArtworkPath,
+                ChannelNameEncoded = channel.Name.Replace(" ", "%20")
```

Comment:
Can `WebUtility.UrlEncode` be used here?

Reply:
WebUtility.UrlEncode will set `+` instead of `%20`

Reply:
This tests successfully for me - the client still requests with %20 when the XML contains `+`.

Reply:
https://learn.microsoft.com/en-us/dotnet/api/system.net.webutility.urlencode?view=net-8.0 `Each ' ' (space) character is converted to a + (plus) character.`

Reply:
Ok, the + sign is well interpreted as well, let's use WebUtility.UrlEncode :D